MapboxSearch/base/src/test/java/com/mapbox/search/base/tests_support/BaseTestDataFactory.kt

```diff
@@ -72,7 +74,7 @@ internal fun createTestBaseRawSearchResult(
 @Suppress("LongParameterList")
 internal fun createTestBaseRequestOptions(
     core: CoreRequestOptions = createTestCoreRequestOptions(),
-    requestContext: SearchRequestContext = SearchRequestContext(CoreApiType.SEARCH_BOX),
+    requestContext: SearchRequestContext = SearchRequestContext(CoreApiType.SBS),
```

Comment:
Why the change to SBS from Search_Box here?

Reply:
Good catch! I was troubleshooting something back then and forgot to revert this temporary change. Updated now.